fix_long_lines.py

```
@@ -0,0 +1,115 @@
+    if modified:
+        with open(file_path, 'w', encoding='utf-8') as f:
+            f.writelines(lines)
+        return True
+    return False
+
+def main():
+    print("🔧 Fixing long lines (E501)...")
+    
+    # Find Python files with E501 issues
+    try:
+        result = subprocess.run(['python3', '-m', 'flake8', '--select=E501', 'src/'], 
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)